api/handler.go

```diff
@@ -383,14 +383,27 @@ func (h *apiHandler) queryTotalUsage() http.HandlerFunc {
 			return
 		}
 
-		query := usage.QuerySpec{
-			From:     from,
-			To:       to,
-			TimeStep: "day",
-			Filter: usage.QueryFilter{
-				UserID:    "",
-				CreatorID: "",
-			},
+		_, ok := r.Context().Value(userIdContextKey).(string)
+		if !ok {
+			respondError(rw, http.StatusInternalServerError, errors.New("request not authenticated"))
+			return
+		}
+
+		isCallerAdmin, ok := r.Context().Value(isCallerAdminContextKey).(string)
```

Comment:
Was this left as a `string` in the end? No need to change, just making sure this shouldn't be `bool`

Reply:
Yes! It was left as a string, will do a pr to change it to bool, I just wanted it to work that day